css/types/color.json

```diff
@@ -0,0 +1,733 @@
+{
+  "css": {
+    "types": {
+      "color": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/color_value",
```

Comment:
I think this should have a "description" so we can call it `<color>`, like the other data types: https://github.com/mdn/browser-compat-data/blob/master/css/types/url.json#L6